### DISCUSSION TITLE
Is there any alternative in Automapper 11 to support what Automapper.Collections been doing?

### DISCUSSION THREAD

**Discussion by shidcordero** [LuckyPennySoftware/AutoMapper#3934] [NA]
Automapper.Collections can have equality comparison for List to List mapping. Hoping there is an alternative for it in Automapper 11 since Automapper.Collections does not support it yet. I might be missing something in the documentation.

**Comment by lbargaoanu** [comment]
https://github.com/AutoMapper/AutoMapper/discussions/3894
